lib/languages.ts

```diff
@@ -725,6 +725,17 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    gimple: {
+        name: 'GIMPLE',
+        monaco: 'nc',
+        extensions: ['.c'],
+        alias: [],
+        logoUrl: 'gimple.svg',
+        logoUrlDark: null,
+        formatter: 'clangformat',
```

Comment:
Is it really going to work correctly with clangformat?

Reply:
Actually, looking at a more complex example maybe not :)